monitor/census.go

```diff
@@ -386,6 +387,7 @@ func InitCensus(nodeType NodeType, version string) {
 	census.mAICurrentLivePipelines = stats.Int64("ai_current_live_pipelines", "Number of live AI pipelines currently running", "tot")
 	census.mAIFirstSegmentDelay = stats.Int64("ai_first_segment_delay_ms", "Delay of the first live AI segment being processed", "ms")
 	census.mAILiveAttempts = stats.Int64("ai_live_attempts", "AI Live stream attempted", "tot")
+	census.mAINumOrchs = stats.Int64("ai_num_orchs_avail", "AI Live number of known orchestrators", "tot")
```

Comment:
would suggest following the metric name convention as mentioned in prometheus docs: https://prometheus.io/docs/practices/naming/

Reply:
so having `_total` as suffix?